test/data.js

```diff
@@ -17,6 +17,38 @@ const createWalletWithChain = async () => {
   return {wallet: wallet, walletKey: walletKey, chainId: chainId}
 }
 
+const itRequires = (version) => {
+  const curVer = process.env.TUPELO_VERSION || "";
+  if (curVer === "" || curVer === "master") {
+    return it;
+  }
+
+  const [curMajor, curMinor, _] = curVer.split(".");
+  const components = version.split(".");
+  const numComponents = components.length;
+  if (numComponents === 0) {
+    return it;
+  }
+
+  if (components[0] > curMajor) {
+    return it.skip;
+  }
+  if (numComponents === 1) {
+    return it;
+  }
+  if (components[1] > curMinor) {
+    return it.skip;
+  }
+  if (numComponents === 2) {
+    return it;
+  }
+  if (components[2] > curMinor) {
```

Comment:
I may be misreading this, but I read this as "is patch > currentMinor"?

Reply:
@brandonwestcott Oops yes, I can fix in a follow-up PR. Thanks!